packages/graphql/src/services/__tests__/graphql.service.spec.ts

```diff
@@ -625,6 +626,28 @@ describe('GraphqlService', () => {
       expect(querySpy).toHaveBeenCalled();
       expect(currentPagination).toEqual({ pageNumber: 3, pageSize: 20 });
     });
+
+    describe("CursorBased related scenarios", () => {
+      describe.each`
+        description           | cursorArgs                    | expectation
```

Comment:
looks like a nice way of describing the test 👍🏻 though I didn't run the test yet, I could imagine how it would look like (like a table I imagine) 😉